cms/djangoapps/contentstore/tasks.py

```diff
@@ -190,7 +190,13 @@ def update_search_index(course_id, triggered_time_isoformat):
         CoursewareSearchIndexer.index(modulestore(), course_key, triggered_at=(_parse_time(triggered_time_isoformat)))
 
     except SearchIndexingError as exc:
-        LOGGER.error(u'Search indexing error for complete course %s - %s', course_id, text_type(exc))
+        error_list = exc.error_list
```

Comment:
gah, I wished that `str(SearchIndexingError)` just included the error list!

Reply:
Yeah, I took a stab at that (and did have it coded up), but then started rabbit-holing on `__str__` vs. `__repr__`, so I punted for now..

Reply:
for sure, not worth rabbit-holing over